puppet/modules/jenkins_node/manifests/unittests.pp

```diff
@@ -70,9 +70,10 @@
   ensure_packages(['python3-virtualenv'])
 
   # nodejs/npm for JavaScript tests
-  if $facts['os']['family'] == 'RedHat' {
+  if $facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] == '9' {
     class { 'nodejs':
-      repo_url_suffix       => '14.x',
+      repo_version          => '18',
+      repo_class            => 'nodejs::repo::dnfmodule',
       nodejs_package_ensure => latest,
```

Comment:
If we rely on RPM packages, can we change this to `present` and instead rely on the `dnf.timer` to update?

Reply:
The reason I say this is that `latest` will purge the dnf cache on every run and that's putting a higher load on mirrors than needed.

Reply:
Fair, I can update that on Monday

Reply:
On a related note: I recall this because it used to be an extremely easy way to overload Satellite.